decidim-debates/app/commands/decidim/debates/admin/create_debate.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    module Admin
+      # This command is executed when the user creates a Debate from the admin
+      # panel.
+      class CreateDebate < Rectify::Command
+        def initialize(form)
+          @form = form
+        end
+
+        # Creates the debate if valid.
+        #
+        # Broadcasts :ok if successful, :invalid otherwise.
+        def call
+          return broadcast(:invalid) if form.invalid?
+
+          create_debate
```

Comment:
Could you also add a notification here to the participatory space followers?

Reply:
Done